src/demo-app/demo-app/demo-app.html

```diff
@@ -25,12 +25,14 @@
       </button>
       <div class="demo-toolbar">
         <h1>Angular Material Demos</h1>
-        <button md-button (click)="toggleFullscreen()" title="Toggle fullscreen">
-          Fullscreen
-        </button>
-        <button md-button (click)="root.dir = (root.dir == 'rtl' ? 'ltr' : 'rtl')" title="Toggle between RTL and LTR">
-          {{root.dir.toUpperCase()}}
-        </button>
+        <div>
+          <button md-button (click)="toggleFullscreen()" title="Toggle fullscreen">
+            Fullscreen
```

Comment:
Since you're incidentally touching this, can you right-align this and make it an md-icon-button with the `fullscreen` icon 😄 ? That's been bothering me forever.